src/types/TrieveSearchAPIResponse.ts

```diff
@@ -1,33 +1,28 @@
 export interface TrieveSearchAPIResponse {
-  score_chunks: ScoreChunk[];
-  total_chunk_pages: number;
+  chunks: ChunkElement[];
+  total_pages: number;
 }
 
-export interface ScoreChunk {
-  metadata: Metadatum[];
+export interface ChunkElement {
+  chunk: ChunkChunk;
   highlights: string[];
   score: number;
 }
 
-export interface Metadatum {
+export interface ChunkChunk {
   id: string;
   link: string;
   qdrant_point_id: string;
   created_at: Date;
   updated_at: Date;
   chunk_html: string;
-  metadata: Metadata;
+  metadata: { [key: string]: string };
```

Comment:
```suggestion
  metadata: object;
```

The value here is not necessarily a string. It can be arbitrary json. 

Mildly complex to handle in our search playground because we have to make sure to handle nesting correctly - https://github.com/devflowinc/trieve/blob/main/frontends/search/src/components/ScoreChunk.tsx#L407-L434

Reply:
resolved in 922f37eeeac4235b98792be42d78d21975403a3e